alembic/versions/5002e75c0604_add_task_parent_to_retain_history_after_.py

```diff
@@ -0,0 +1,21 @@
+"""Add task parent to retain history after split
+
+Revision ID: 5002e75c0604
+Revises: 33e34b3beaa3
+Create Date: 2016-05-10 22:37:31.383527
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '5002e75c0604'
+down_revision = '33e34b3beaa3'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade():
+    op.add_column('task', sa.Column('parent_id', sa.Integer(), nullable=True))
+
+def downgrade():
```

Comment:
I probably won't have time to tackle that for a few days, so if you want to go ahead and work on that you are welcome to. Though I do agree it shouldn't be too difficult to do.